tests/pytests/test_vecsim.py

```diff
@@ -653,3 +653,21 @@ def test_single_entry(env):
                'RETURN', '0',
                'PARAMS', 2, 'vec_param', vector.tobytes()).equal([1L, '0'])
 
+def test_wrong_vector_size(env):
+    conn = getConnectionByEnv(env)
+    dimension = 128
+    
+    vector = np.random.rand(1+dimension).astype(np.float32)
+    conn.execute_command('HSET', '0', 'v', vector[:dimension-1].tobytes())
+    conn.execute_command('HSET', '1', 'v', vector[:dimension].tobytes())
+    conn.execute_command('HSET', '2', 'v', vector[:dimension+1].tobytes())
+
+    conn.execute_command('FT.CREATE', 'idx', 'SCHEMA', 'v', 'VECTOR', 'HNSW', '6', 'TYPE', 'FLOAT32', 'DIM', dimension, 'DISTANCE_METRIC', 'L2')
+
+    vector = np.random.rand(1+dimension).astype(np.float32)
+    conn.execute_command('HSET', '3', 'v', vector[:dimension-1].tobytes())
+    conn.execute_command('HSET', '4', 'v', vector[:dimension].tobytes())
+    conn.execute_command('HSET', '5', 'v', vector[:dimension+1].tobytes())
```

Comment:
Consider checking the error message as well for ids `3` and `5`, as you did for `ft.search`

Reply:
there is no error message, as `HSET` command is legal even if we can't index it.

Reply:
@ashtul Isn't there a way of retrieving error message in such case? `ft.info` only contains the number of failures right?

Reply:
@alonre24 No. there is no way to check it while inserting docs.
Instead, Guy added a test for `hash_indexing_failures` parameter

Reply:
@GuyAv46 if there is no way to test the error message upon insertion, please remove it/log it with low severity

Reply:
OK